scripts/build-docker

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+ARCH=$(go env GOARCH)
```

Comment:
okay, I think once we publish the release, we can pull it from release page instead of building them. Let me your opinion on this?